kv/config.go

```diff
@@ -22,15 +22,17 @@ package kv
 
 // OverrideConfiguration is the config for OverrideOptions.
 type OverrideConfiguration struct {
-	Zone        string `yaml:"zone"`
-	Environment string `yaml:"environment"`
-	Namespace   string `yaml:"namespace"`
+	Zone              string `yaml:"zone"`
+	Environment       string `yaml:"environment"`
+	Namespace         string `yaml:"namespace"`
+	WatchWithRevision int64  `yaml:"watchWithRevision"`
```

Comment:
The Override config/options are managed by us and not specific to the data store, but the revision is pretty specific to the data store, for example etcd supports it but many other store don't, can we not have this field here and only keep it in the etcd specific kv options?

Reply:
I agree that it's pretty implementation-specific, but I wanted to provide flexibility to set this on a per-store basis instead of having to set it on every store created from a client. If we're fine with this being set at the client level and every store created from the client inheriting it I'll change it. Do you think we need the flexibility or can set this on every store from a single client?